<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>以往日记</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="3">
          <widget class="QLabel" name="labeltouxiang">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>121</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>121</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelwangming">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>84</width>
             <height>36</height>
            </size>
           </property>
           <property name="toolTip">
            <string>网名</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelzhanghao">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>84</width>
             <height>37</height>
            </size>
           </property>
           <property name="toolTip">
            <string>账号</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelgeqian">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>84</width>
             <height>36</height>
            </size>
           </property>
           <property name="toolTip">
            <string>个性签名</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QListWidget" name="listWidget">
           <property name="maximumSize">
            <size>
             <width>211</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labeltianqi">
             <property name="text">
              <string>天气：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labeltianqi1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelxinqing">
             <property name="text">
              <string>心情：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelxinqing1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionxieriji"/>
   <addaction name="actionshuxin"/>
   <addaction name="actionpeizhi"/>
   <addaction name="actionguanyu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionxieriji"/>
    <addaction name="actionshuxin"/>
    <addaction name="separator"/>
    <addaction name="actionpeizhi"/>
    <addaction name="separator"/>
    <addaction name="actionguanyu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionxieriji">
   <property name="text">
    <string>写日记</string>
   </property>
  </action>
  <action name="actionshuxin">
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionpeizhi">
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="actionguanyu">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
